### PR TITLE
PR for Product Filter 

### PR DESCRIPTION
**Pull request by manismk**
- Added filter to product page
- Sort by Price, Rating and category.
- leave product image for now will update later
- [link](https://deploy-preview-2--jstees-react.netlify.app/)

### CODE REVIEW

File: src/Pages/Products/components/Category.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      type: "CATEGORY_CHANGE",
+      payload: e.target.value,
+    });
+  };
+
+  return (
+    <div className="filter--category m-v-1">
+      <h4 className="heading--4">Category</h4>
+      <div className="input--primary">
+        <input
```
Comment (xsaurabhsuthar) [inline src/Pages/Products/components/Category.jsx]: You can extract the checkbox input as a component.

File: src/Pages/Products/components/Filters.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <>
+      <button className="filter--button">
+        Filter <i className="fas fa-arrow-right"></i>
+      </button>
+      <div className="overlay hide"></div>
+      <section className="filter--section">
+        <button className="filter--close">&times;</button>
+        <div className="filter--header m-v-1">
+          <h3 className="heading--3">Filter</h3>
+          <button
```
Comment (xsaurabhsuthar) [inline src/Pages/Products/components/Filters.jsx:16]: You can also extract the button.

File: src/Pages/Products/components/Ratings.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+          value="4"
+          checked={state.rating === "4" ? true : false}
+          onChange={(e) => {
+            changeHandler(e);
+          }}
+        />
+        <label htmlFor="4star">4 star & above</label>
+      </div>
+      <div className="input--primary">
+        <input
```
Comment (xsaurabhsuthar) [inline src/Pages/Products/components/Ratings.jsx]: Radio input too.

**Review by xsaurabhsuthar — APPROVED** [review]
Everything looks clean. Good to merge 🚀

**Review by muskaanshah — APPROVED** [review]
Looks good, maybe you can extract checkboxes and radio as a component since it's being used repeatedly

**Comment by manismk** [comment]
Thanks, @muskaanshah and @Launchpad5682  for the review. I have converted it to components

**Pull request merged**